packages/ember-data/lib/serializers/json-api-serializer.js

```diff
@@ -58,12 +58,12 @@ var dasherize = Ember.String.dasherize;
               "data": [
                 {
                   "id": "3",
-                  "type": "players"
+                  "type": "player"
```

Comment:
ED uses camelCased attributes and singular types internally, regardless of what adapter/serializer you're using.

When you're using the JSON API adapter/serializer we want users to be able to use the examples available on jsonapi.org and have it just work. Most users never have to care about the internal format since the serializer handles the work for them.

This is documented in the guides, http://guides.emberjs.com/v2.0.0/models/pushing-records-into-the-store/